### PR TITLE
Remove backward ops for cuDNN transposed convolution

### PR DESCRIPTION
**Pull request by jbschlosser**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #69902
* #69901

Differential Revision: [D33093795](https://our.internmc.facebook.com/intern/diff/D33093795)

### CODE REVIEW

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33093795).

File: torch/csrc/autograd/FunctionsManual.cpp [hunk 1]
```diff
@@ -4763,6 +4763,22 @@ std::tuple<Tensor, Tensor> _cudnn_convolution_backward(
   return result;
 }
 
+std::tuple<Tensor, Tensor> _cudnn_convolution_transpose_backward(
```
Comment (albanD) [inline torch/csrc/autograd/FunctionsManual.cpp:4766]: Can we re-use the op from above with an extra boolean argument for transpose?

**Review by albanD — COMMENTED** [review]
I think we should re-use the other cudnn backward op. Otherwise looks good.

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33093795).

**Review by albanD — APPROVED** [review]
SGTM

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33093795).

**Review by albanD — APPROVED** [review]
Good catch

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33093795).

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33093795).

**Comment by jbschlosser** [comment]
@jbschlosser has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33093795).

**Pull request closed without merging**